products.css

```diff
@@ -0,0 +1,53 @@
+
+
+#fruits {
```

Comment:
what you actually want to flex is the li elements, their parent container is ul.  try that here instead of fruits